File: web/test_loader.dart
import 'dart:html';

import 'dart:async';
import 'dart:convert';

import 'package:beer_run/game.dart';
import 'package:beer_run/loader.dart';


void main(List<String> args) {

	Map configData = JSON.decode(querySelector('div#config').innerHtml);
    GameConfig config = new GameConfig(configData);
    Map applicationConfig = config.get('application');

	CdnLoader cdnLoader = new CdnLoader(applicationConfig['assets']['cdn_hosts'], 1);
	cdnLoader.loadManifest().then((var _) => window.alert("done done done"));
}

File: web/test_loader.php
<?php

include '../server/init.php';

?>

<html>
<head>
	<title>Test the Loader</title>
</head>
<body>
	<script type="application/dart" src="/test_loader.dart.js"></script>
	<script src="/packages/browser/dart.js"></script>
  	<div id="config"><?=$config->asJsonString();?></div>

</body>
</html>